README.md

```diff
@@ -125,3 +125,7 @@ yarn
 yarn build
 yarn publish
 ```
+
+---
+
+## [Changelog](https://docs.google.com/document/d/1G7tknW4cSXJvF00TTHjzCH-4-4dvR1k6G8MQMNba5M8/edit?usp=sharing)
```

Comment:
Can we use this link instead pls:

https://docs.google.com/document/d/e/2PACX-1vR5NzVS2msrCMZxlcfBgAT-Y8kAypeKqH_WBeNiwVTmyEzLZvJBWrKrz_966-d3jumwIBi94IXGT6Wp/pub

It's the "publishable" version of the same google doc